src/test/java/com/rultor/agents/github/CommentsTagTest.java

```diff
@@ -93,6 +99,86 @@ public void duplicatesRelease() throws Exception {
     }
 
     /**
+     * CommentsTag cannot release an older version.
+     * @throws IOException In case of error.
+     */
+    @Test
+    public void rejectsOldRelease() throws IOException {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        final Agent agent = new CommentsTag(repo.github());
+        repo.releases().create("1.0");
+        repo.releases().create("2.0");
+        repo.releases().create("3.0-b");
+        final Talk talk = CommentsTagTest.talk(issue, "1.5");
+        agent.execute(talk);
+        final Comment.Smart response = new Comment.Smart(
+            repo.issues().get(1).comments().get(1)
+        );
+        MatcherAssert.assertThat(
+            response.body(),
+            Matchers.containsString("version tag is too low")
+        );
+    }
+
+    /**
+     * CommentsTag can validate or reject a version.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void canValidateVersion() throws Exception {
+        Assert.assertTrue(
```

Comment:
@gumbelmj Let's get rid of this test and replace it with one where you invoke `CommentsTag.execute` with a talk like `CommentsTagTest.talk(issue, "beta")` for example.